### PR TITLE
M20-52: Full takeover of Ghostery rewards 

### PR DESCRIPTION
**Pull request by aleksandr-cliqz**
https://cliqztix.atlassian.net/browse/M20-52
@luciancor @christophertino @sammacbeth @olpa-cliqz 

Description:
the main idea to show myoffrz-rewards instead of ghostery-rewards in ghostery-extension.
Navigation-extension: we treat ghostery as whitelabel extension (see e.g. chip extension)
Ghostery-extension: we reuse in content-script cliqz-bundle (with myoffrz-rewards) and show list of rewards in ghostery-cc; we will use iframe to show list of rewards.

Locales:
There are around 45 keys in the pull-request. Some of them new, some renamed to follow convension (prefix myoffrz_)  English: all keys, Poland: around 20 renamed, other countries: will be addressed in next prs 

* [ ] Have you followed the guidelines in [CONTRIBUTING.md](../CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [x] Have you added an explanation of what your changes do?
* [x] Does your submission pass tests?
* [x] Did you lint your code prior to submission?

### CODE REVIEW

File: src/background.js [hunk 1]
```diff
@@ -726,6 +654,20 @@ function reportCliqzOffer(message) {
  * @return {boolean}            denotes async (true) or sync (false)
  */
 function onMessageHandler(request, sender, callback) {
+	if (request.module === 'offers-banner' && request.action === 'send') {
```
Comment (luciancor) [inline src/background.js]: @christophertino this does not seem great here, but not sure where else fits better. any suggestion?

File: app/panel/components/Panel.jsx [hunk 2]
```diff
@@ -142,10 +142,6 @@ class Panel extends React.Component {
 				reload: true
 			});
 		}
-
-		if (panel.enable_offers && panel.unread_offer_ids.length > 0) {
```
Comment (aleksandr-cliqz) [inline app/panel/components/Panel.jsx]: plz look at this

File: src/background.js (hunk 1 above)
Comment (aleksandr-cliqz) [reply src/background.js]: just use ```import { sendRewardMessage } from '../utils/msg';```

File: app/panel/components/Panel.jsx (hunk 2 above)
Comment (christophertino) [reply app/panel/components/Panel.jsx]: we can remove this

File: src/classes/Metrics.js [hunk 3]
```diff
@@ -250,13 +247,6 @@ class Metrics {
 			case 'rewards_on':
 				this._sendReq(type, ['all', 'daily']);
 				break;
-			case 'rewards_first_learn_more':
```
Comment (aleksandr-cliqz) [inline src/classes/Metrics.js]: let's remove too ```case 'engaged_offer':```

**Comment by christophertino** [comment]
Remove unused package:
`yarn remove react-shadow-dom-retarget-events`

**Comment by aleksandr-cliqz** [comment]
lets keep only en translations

**Comment by aleksandr-cliqz** [comment]
WIP: do not merge it yet plz; checking some stuff

**Comment by christophertino** [comment]
@aleksandr-cliqz Looks good to me. Once we get the new browser-core build added we can merge it up.

**Comment by aleksandr-cliqz** [comment]
@christophertino could you help me plz with continuous-integration/travis-ci/pr ?
in my last commit I added new version of Cliqz package 
seems I need somehow to update yarn.lock too

**Comment by aleksandr-cliqz** [comment]
@christophertino 
`Once we get the new browser-core build added we can merge it up.`
done :)

**Pull request merged**